<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Печать этикеток 2.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.25, y2:0.25, stop:0 rgba(245, 245, 245, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="leftWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#3e3e3e;&quot;&gt;Новое задание&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listView">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="rightWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#3e3e3e;&quot;&gt;Сохраненные задания&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(250, 250, 250);
border-right: 2px solid rgb(210, 210, 210);
border-bottom: 2px solid rgb(210, 210, 210);
border-radius: 10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>454</width>
            <height>453</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border: none;</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>65</height>
          </size>
         </property>
         <widget class="QToolButton" name="toolButton">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>10</y>
            <width>52</width>
            <height>52</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="btnIcons.qrc">
            <normaloff>:/btnIcon/GearAndKey.bmp</normaloff>:/btnIcon/GearAndKey.bmp</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="autoRepeat">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="btnIcons.qrc"/>
 </resources>
 <connections/>
</ui>
